build.gradle.kts

```diff
@@ -33,6 +33,8 @@ kotlin {
     linuxArm64()
     linuxArm32Hfp()
     linuxX64()
+    linuxMips32()
+    linuxMipsel32()
```

Comment:
```suggestion
    linuxMips32()
    linuxMipsel32()
    linuxX64()
```
(The current list is alphabetized)

Reply:
Well, kinda. I see I put 32 after 64 above 🙈

Reply:
Done (if I got it correctly)